### ISSUE TITLE
Broken code folding 

### ISSUE THREAD

**Issue by farski** [redhat-developer/vscode-yaml#427]
## Describe the bug
I'm having an issue when this extension is enabled where it is folding sections of code beyond their boundaries.

## Expected Behavior

For a file like [this](https://github.com/PRX/Infrastructure/blob/master/stacks/publish.prx.org.yml), I would expect folding a level 2 entity (e.g., `DeployBucketPolicy`, to only fold up until the end of that entity.

![image](https://user-images.githubusercontent.com/3483/107400975-637fe500-6ad0-11eb-9627-3f5b40c7025d.png)

Without this extension enabled, this happens as expected.

![image](https://user-images.githubusercontent.com/3483/107401105-8b6f4880-6ad0-11eb-805f-922f0b889754.png)

## Current Behavior

Currently, when this extension is enabled, folding `DeployBucketPolicy` folds the lower-level `Outputs` into itself as well. In this case, the next level 2 entity is visible, but in many other cases where this issue also presents itself, entire entities can be hidden by folding a sibling entity.

![image](https://user-images.githubusercontent.com/3483/107400991-6975c600-6ad0-11eb-85b4-2ad0eae66550.png)

## Steps to Reproduce

1. Open a [YAML file](https://github.com/PRX/Infrastructure/blob/master/stacks/publish.prx.org.yml)
2. Fold is at level 2 with the extension turned on
3. Compare the results with the extension disabled

## Environment
- [ ] Windows
- [x] Mac
- [ ] Linux
- [ ] other (please specify)

**Comment by sgtoj** [comment]
Duplicate of #422. It is should be resolved with v0.15. That release scheduled sometime today. I am waiting on it, as well as dozens of others at my company.

**Comment by cavo789** [comment]
Works fine! Thanks
